apps/emqx_authn/src/emqx_authn.appup.src

```diff
@@ -1,5 +1,117 @@
 %% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{<<".*">>,[]}],
-  [{<<".*">>,[]}]}.
+  [{"0.1.6",
```

Comment:
FYI: the open source version is not required to support hot upgrade. So, many changes were not written in appup.src.  but I think it's okay to add them now :thinking:

Reply:
We decided better remove them, because many unrelated apps are unnecessarily tocuhed